js/src/elements/form-behavior.ts

```diff
@@ -364,6 +366,16 @@ export class FormBehaviorElement extends HTMLElement {
 		return null;
 	}
 
+	validateGroupRequired(input: HTMLValueElement, label: string): string | null {
+		const group = getField(input);
+		const inputs = Array.from(group.querySelectorAll<HTMLInputElement>('input'));
+		const selected = inputs.some(input => input.checked);
+		if (!selected) {
+			return `${this.locStrings.inputGroupRequired.replace('{inputLabel}', label)}`;
```

Comment:
Super nit, feel free to ignore.

Template string shouldn't be required here.
```suggestion
			return this.locStrings.inputGroupRequired.replace('{inputLabel}', label);
```

Reply:
Adjusted the other validation methods to remove the template string syntax